CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## [1.2.0](https://github.com/theforeman/puppet-pulpcore/tree/1.2.0) (2020-08-06)
+
+[Full Changelog](https://github.com/theforeman/puppet-pulpcore/compare/1.1.0...1.2.0)
+
+**Implemented enhancements:**
+
+- Fixes [\#29895](https://projects.theforeman.org/issues/29895) - add pulp\_deb plugin [\#112](https://github.com/theforeman/puppet-pulpcore/pull/112) ([m-bucher](https://github.com/m-bucher))
+- Use pulpcore-manager [\#111](https://github.com/theforeman/puppet-pulpcore/pull/111) ([ekohl](https://github.com/ekohl))
+- Refs [\#30057](https://projects.theforeman.org/issues/30057) - Configure Pulpcore Worker Count [\#100](https://github.com/theforeman/puppet-pulpcore/pull/100) ([wbclark](https://github.com/wbclark))
+
+**Fixed bugs:**
+
+- Add the GPL license text [\#109](https://github.com/theforeman/puppet-pulpcore/pull/109) ([ekohl](https://github.com/ekohl))
+
+**Closed issues:**
+
+- Figure out how to make pulp-worker count configurable [\#37](https://github.com/theforeman/puppet-pulpcore/issues/37)
```

Comment:
I've added the skip-changelog label since there's already a PR that implements this, making this line redundant. In general issues don't really show up in our changelogs and I've considered just skipping them unconditionally.

Reply:
Thanks, I reviewed the PRs but missed the closed issues. I'll update the changelogs to reflect this.